charts/kong/templates/_helpers.tpl

```diff
@@ -1583,6 +1582,12 @@ resource roles into their separate templates.
   - get
   - patch
   - update
+- apiGroups:
+    - ""
+  resources:
+    - namespaces
+  verbs:
+    - list
```

Comment:
```suggestion
    - get
    - list
    - watch
```

Reply:
Actually I believe we should add that in the `CluterRole` rules rather than in `Role`. PTAL [f412d1e](https://github.com/Kong/charts/pull/974/commits/f412d1ef40fcc7cf396c7b50bd9be1b4f689a6df)